volcli/volcli.go

```diff
@@ -315,7 +335,7 @@ func UseGet(ctx *cli.Context) {
 // UseTheForce deletes the use entry from etcd; useful for clearing a
 // stale mount.
 func UseTheForce(ctx *cli.Context) {
-	if len(ctx.Args()) != 2 {
+	if len(ctx.Args()) != 1 {
```

Comment:
I think the `ArgUsage` string update for `get` and `force-remove` commands is missing above. Sorry, I could post comment there as github only let's me comment on changed lines.